From 177b5d2e3f0f51d7a7fb37c2bf8c71c9c48e8d72 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Mon, 18 Apr 2016 15:34:24 -0500
Subject: [PATCH 09/23] ARM: dts: modify reset enet phy signal

Reasign GPIO reset phy signal. It used to be on CABC_EN.
Disabled fec2 driver.

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/boot/dts/imx6sxscm-evb.dtsi | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sxscm-evb.dtsi b/arch/arm/boot/dts/imx6sxscm-evb.dtsi
index e7feead..b1756b7 100644
--- a/arch/arm/boot/dts/imx6sxscm-evb.dtsi
+++ b/arch/arm/boot/dts/imx6sxscm-evb.dtsi
@@ -13,6 +13,15 @@
 	chosen {
 		stdout-path = &uart3;
 	};
+
+	hannstar_cabc {
+		compatible = "hannstar,cabc";
+
+		lvds0 {
+			gpios = <&gpio4 18 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
 };
 &uart3 {
 	pinctrl-names = "default";
@@ -167,6 +176,7 @@
 				MX6SX_PAD_RGMII1_RX_CTL__ENET1_RX_EN	0x3081
 				MX6SX_PAD_QSPI1A_DATA0__GPIO4_IO_16	0x80000000
 				MX6SX_PAD_QSPI1B_DATA1__GPIO4_IO_25 	0x80000000
+				MX6SX_PAD_QSPI1B_DATA2__GPIO4_IO_26     0x80000000
 				MX6SX_PAD_ENET2_RX_CLK__ENET2_REF_CLK_25M	0x91
 			>;
 		};
@@ -177,8 +187,12 @@
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet1>;
-	pinctrl-assert-gpios = <&gpio4 16 GPIO_ACTIVE_HIGH>, <&gpio4 25 GPIO_ACTIVE_LOW>;
+	pinctrl-assert-gpios = <&gpio4 16 GPIO_ACTIVE_HIGH>, <&gpio4 25 GPIO_ACTIVE_LOW>, <&gpio4 26 GPIO_ACTIVE_HIGH>;
 	phy-mode = "rgmii";
 	status = "okay";
 };
 
+&fec2 {
+	status = "disabled";
+};
+
-- 
1.9.1

